cmds/stage

```diff
@@ -124,10 +114,33 @@ stage_repository() {
         img_dst_label="${entry[4]}"
         img_dst_mnt="${entry[5]}"
 
+        case "${img_type}" in
+            tar.bz2) img_format="tarbz2";;
+            ext3.gz) img_format="ext3gz";;
+            vhd.gz)
```

Comment:
`vhd.gz` still needs a globing for `uivm-rootfs.ext3.vhd.gz`, `ndvm-rootfs.ext3.disk.vhd.gz` and `syncvm-rootfs.ext3.vhd.gz`.